Ticket - 35437 Copy Machine, Edit and Save -> Standards Tab showing previous machine

sewa67 opened the pull request:
Please review this change and merge into development branch 
Files - viewers/mach.w

sewa67 committed:
Ticket - 35437 Copy Machine, Edit and Save -> Standards Tab showing previous machine
Files - viewers/mach.w

brad-advantzware reviewed (changes requested):
@sewa67 The only change I see is the Obsolete changed to Inactive.  The main defect as originally reported is still there.

sewa67 committed:
Ticket - 35437 Copy Machine, Edit and Save -> Standards Tab showing previous machine
Files - pc/v-prdh.w, pc/b-prddj2.w, jc/v-job.w, jc/jobstds.p, jc/chkrebld.p, jc/b-jobmch.w,
fg/m-mchexp.w, est/d-estop.w, est/b-estop.w, ce/probe.w, ce/d-estop.w, ce/b-estop.w,
browsers\probe.w
